api/v1beta1/clusterclass_types.go

```diff
@@ -965,20 +990,23 @@ type PatchSelectorMatchMachinePoolClass struct {
 	// names selects templates by class names.
 	// +optional
 	// +kubebuilder:validation:MaxItems=100
+	// +kubebuilder:validation:items:MaxLength=256
 	Names []string `json:"names,omitempty"`
 }
 
 // JSONPatch defines a JSON patch.
 type JSONPatch struct {
 	// op defines the operation of the patch.
 	// Note: Only `add`, `replace` and `remove` are supported.
+	// +kubebuilder:validation:MaxLength=8
 	Op string `json:"op"`
```

Comment:
Should probably be an enum in the future?

Reply:
Ah got your point that we don't need maxlength if we add the enum validation. Added it now already (other operations lead to failures already anyway)